pyrealm/pmodel/subdaily.py

```diff
@@ -267,7 +267,8 @@ def __init__(
             "co2",
             "patm",
             "vpd",
-        ] + optimal_chi_class.requires
+            *optimal_chi_class.requires,
```

Comment:
This seems like a strange change. Is it separate to the edits to support ruff?

Reply:
Yeah - it's this rule: https://docs.astral.sh/ruff/rules/collection-literal-concatenation/

I believe them, but it puzzled me too!

Reply:
yeah, its not super obvious is it!